torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -437,25 +482,28 @@ def check_is_root(self) -> bool:
 
     @staticmethod
     def fsdp_modules(
-        module: nn.Module, root_only: bool = False
+        module: nn.Module,
+        root_only: bool = False,
     ) -> List["FullyShardedDataParallel"]:
         """
-        Helper function to return all nested FSDP instances, including self.
+        Returns all nested FSDP instances, possibly including ``module`` itself
+        and only including FSDP root modules if ``root_only=True``.
 
         Args:
-            module: the root module. This module does not have to be a FSDP module.
-            root_only: whether to return only root FSDP modules (default: False).
+            module (torch.nn.Module): Root module, which may or may not be an
+                ``FSDP`` module.
+            root_only (bool): Whether to return only FSDP root modules.
+                (Default: ``False``)
 
         Returns:
-            fsdp_modules: the FSDP modules that are nested in the input module.
+            List[FullyShardedDataParallel]: FSDP modules that are nested in
+            the input ``module``.
         """
-        fsdp_modules = []
-        for sub_module in module.modules():
-            if isinstance(sub_module, FullyShardedDataParallel):
-                if not root_only or sub_module.check_is_root():
-                    fsdp_modules.append(sub_module)
-
-        return fsdp_modules
+        return [
+            submodule for submodule in module.modules()
+            if isinstance(submodule, FullyShardedDataParallel) and
+            (not root_only or submodule.check_is_root())
+        ]
```

Comment:
is this a stylistic refactor?

Reply:
Yup, my bad on this. I added something logically at one point and reformatted but then removed the addition.